src/lyrics.py

```diff
@@ -12,33 +12,29 @@ def get_lyrics(song_name):
 
     song_name += ' metrolyrics'
     name = quote_plus(song_name)
-    hdr = {'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.11'
-           '(KHTML, like Gecko) Chrome/23.0.1271.64 Safari/537.11',
-           'Accept-Language': 'en-US,en;q=0.8',
+    hdr = {'User-Agent': 'Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:52.0) Gecko/20100101 Firefox/52.0',
+           'Accept-Encoding': 'gzip, deflate, br',
```

Comment:
On my side the request is not decoded correctly when enabling brötli.
Everything works fine again when changing this line to:
`'Accept-Encoding': 'gzip, deflate',`